algolia/package.json

```diff
@@ -6,18 +6,19 @@
     "start": "node ./bin/www"
   },
   "dependencies": {
+    "async": "~2.4.0",
     "body-parser": "~1.17.1",
     "cookie-parser": "~1.4.3",
     "debug": "~2.6.3",
     "express": "~4.15.2",
+    "glob": "^7.1.2",
```

Comment:
This is the only addition. The others have just been rearranged alphabetically by npm.